src/apps/vpn/appfeaturelist.h

```diff
@@ -53,6 +53,13 @@ FEATURE(freeTrial,              // Feature ID
         QStringList(),          // feature dependencies
         FeatureCallback_false)  // Disabled while we rethink free trials
 
+FEATURE(gleanDebugViewTag,       // Feature ID
```

Comment:
Since this is not a feature, but a configuration value it feels incorrect to use a feature flag to turn it on or off. In my opinion it makes more sense to have a setting value that holds this. And holds it as a string instead of a boolean so that each tester can set it differently.